app/database/result_store_propagate_validated_at_integration_test.go

```diff
@@ -195,7 +195,7 @@ func TestResultStore_Propagate_Categories_SetsValidatedAtToMaxOfValidatedAtsOfCh
 		UpdateColumn("category", "Validation").Error())
 
 	assert.NoError(t, itemStore.Where("id=1").Updates(map[string]interface{}{
-		"type": "Course",
+		"type": "Task",
```

Comment:
Have you checked it was not a Task already? Otherwise your loose a bit a part of the goal of the test.

Reply:
I'm not sure what this is for ? Where does the item with id=1 come from ?

Reply:
For unit tests, data is loaded from fixture files (cfr `db := testhelpers.SetupDBWithFixture("results_propagation/_common", "results_propagation/validated_at")` on top of the test)... so loaded from what is in the `testdata` dir from the test directory.

Reply:
Ok. I'm not sure what to do with this line. The type is not defined within the fixtures files, but the test still works without this line.

Reply:
Ok I thought it was a test for the item get/update... but it is a test for propagation, so the courses behave like tasks, so it is better to keep as you did. 
Sorry, my initial remark was useless here.